great_expectations/metrics/batch/row_count.py

```diff
@@ -3,8 +3,8 @@
 from great_expectations.metrics.metric_results import MetricResult
 
 
```

Comment:
I renamed this file, because I think we will be adding much more logic to each of these classes over time. Happy to discuss if we want to do that now.